pkg/nodemanager/nodemanager_test.go

```diff
@@ -665,6 +759,123 @@ func TestNodeProvidedIPAddresses(t *testing.T) {
 	assert.Equal(t, "10.0.0.1", updatedNodes[0].Status.Addresses[0].Address, "Node Addresses not correctly updated")
 }
 
+func Test_reconcileNodeLabels(t *testing.T) {
+	testcases := []struct {
+		name           string
+		labels         map[string]string
+		expectedLabels map[string]string
+		expectedErr    error
+	}{
+		{
+			name:           "no labels",
+			labels:         map[string]string{},
+			expectedLabels: map[string]string{},
+			expectedErr:    nil,
+		},
+		{
+			name: "requires reconcile",
+			labels: map[string]string{
+				v1.LabelZoneFailureDomainStable: "foo",
+				v1.LabelZoneRegionStable:        "bar",
+				v1.LabelInstanceTypeStable:      "the-best-type",
+			},
+			expectedLabels: map[string]string{
+				v1.LabelZoneFailureDomain:       "foo",
```

Comment:
the code comments for these consts seems to indicate that we shouldn't use them, should we be using the other names instead  (eg `LabelFailureDomainBetaZone` in this case)?

https://github.com/kubernetes/api/blob/master/core/v1/well_known_labels.go#L32

Reply:
Yep, good catch, can fix that